ITGuru.FourWheels/ITGuru.FourWheels.Data/DataModels/Customer.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace ITGuru.FourWheels.Data.DataModels
+{
+    public class Customer : Person
+    {
+        public string Phone;
+        public string Email;
```

Comment:
Should we maybe add the implementation of Soft deleting?

Could be a `Property` of type `bool` called Active, default equals to `true`.